docs/_templates/autosummary/class.rst

```diff
@@ -27,9 +26,10 @@
    .. rubric:: {{ _('Attributes') }}
 
    .. autosummary::
-      :inherited-members:
```

Comment:
This was not working anyway, but can anyone please confirm if the level of documented objects is sufficient without this option?